includes/CodingStandards/GoogleSiteKit/Sniffs/Organizational/CommentTagsCorrectlyGroupedSniff.php

```diff
@@ -0,0 +1,216 @@
+<?php
+/**
+ * Docblock comment tags should be grouped.
+ *
+ * @since n.e.x.t
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace PHP_CodeSniffer\Standards\GoogleSiteKit\Sniffs\Semantic;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Comment Tags Correctly Grouped Sniff.
+ *
+ * @since n.e.x.t
+ */
+class CommentTagsCorrectlyGrouped implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+		return array( T_DOC_COMMENT_OPEN_TAG );
+	}
+
+	/**
+	 * Processes this sniff, when one of its tokens is encountered.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcs_file The current file being checked.
+	 * @param int                         $stack_ptr  The position of the current token in the
+	 *                                                stack passed in $tokens.
+	 * @return void
+	 */
+	public function process( File $phpcs_file, $stack_ptr ) {
+		$tokens        = $phpcs_file->getTokens();
+		$comment_end   = $phpcs_file->findNext( T_DOC_COMMENT_CLOSE_TAG, ( $stack_ptr + 1 ) );
+		$comment_start = $tokens[ $comment_end ]['comment_opener'];
+
+		// List of @ comment types to check.
+		$doc_comment_tags = array(
+			'@since',
+			'@deprecated',
+			'@access',
+			'@static',
+			'@global',
+			'@var',
+			'@param',
+			'@return',
+		);
+
+		// List how the @ tags should be grouped.
+		$doc_comment_tags_groups = array(
+			array(
+				'@since',
+				'@deprecated',
+				'@access',
+				'@static',
+			),
+			array(
+				'@global',
+			),
+			array(
+				'@var',
+				'@param',
+				'@return',
+			),
+		);
```

Comment:
This does not fully implement the ACs; grouping should be slightly different between property docs and function/method docs; in particular: `@var` should only be relevant for property doc comments, and for those type of comments there should be just a single group with everything (most tags, e.g. `@param` and `@return`, are irrelevant for that kind of comment anyway).